modules/swagger-codegen/src/main/resources/Java/libraries/retrofit/CollectionFormats.mustache

```diff
@@ -0,0 +1,95 @@
+package {{invokerPackage}};
+
+import java.util.Arrays;
+import java.util.List;
+
+public class CollectionFormats {
+    
+    public static class CSVParams {
+
+        protected List<String> params;
+        
+        public CSVParams() {
+        }
+        
+        public CSVParams(List<String> params) {
+            this.params = params;
+        }
+        
+        public CSVParams(String... params) {
+            this.params = Arrays.asList(params);
+        }
+
+        public List<String> getParams() {
+            return params;
+        }
+
+        public void setParams(List<String> params) {
+            this.params = params;
+        }
+        
+        @Override
+        public String toString() {
+            return String.join(",", params);
```

Comment:
@cbornet  Shall we use the join method in StringUtil.java instead as the static method `join` in the String class is only available in Java8 but not Java7 (which is the minimum Java requirement for Retrofit)?